Model/src/GamePackage/Player.java

```diff
@@ -1,4 +1,265 @@
 package GamePackage;
 
+import java.util.ArrayList;
+
+import BoardPackage.BoardSquare;
+import BoardPackage.PropertySquare;
+import CardPackage.Card;
+
+/**********************************************************************
+ * The Player class will keep track of each player's money, properties,
+ * and other stats.
+ * 
+ * @author Dustin Ecker
+ * @version 2/11/2018
+ *********************************************************************/
 public class Player {
+	
+	/** Player's name */
+	private String displayName;
+	
+	/** Player's token */
+	private String token;
```

Comment:
Should the token the player uses be final?

Reply:
I agree

Reply:
same thing again, I though we where having different tokens.

Reply:
Once the token is chosen it wont change. Right?